R/tm_a_pca.R

```diff
@@ -240,7 +262,34 @@ ui_a_pca <- function(id, ...) {
               choices = args$plot_choices,
               selected = args$plot_choices[1]
             ),
-            ui_transform_teal_data(ns("decorate"), transformators = args$decorators)
+            conditionalPanel(
+              condition = sprintf("input['%s'] == 'Elbow plot'", ns("plot_type")),
+              ui_decorate_teal_data(
+                ns("d_elbow_plot"),
+                decorators = subset_decorators("elbow_plot", args$decorators)
+              )
+            ),
+            conditionalPanel(
+              condition = sprintf("input['%s'] == 'Circle plot'", ns("plot_type")),
+              ui_decorate_teal_data(
+                ns("d_circle_plot"),
+                decorators = subset_decorators("circle_plot", args$decorators)
```

Comment:
This is minor but my R-instinct tells me it would be more R-ish if `subset_decorators` would take decorators as a first argument, and then names as the second.

Reply:
Sounds nice!

To keep with that R-like, I'm renaming it to `select_decorators()`.. I'm regretting the name for days :grin: